modules/ROOT/pages/bind-api-configure-service-mesh-CRD.adoc

```diff
@@ -0,0 +1,134 @@
+. Click an instance name and copy the API ID that is displayed. 
+Ensure that you save this API ID to a text editor.
++
+image::api-id.png[]
++
+. To retrieve the service name from Kubernetes, with which you want to bind the API, run the command:
++
+`kubectl get svc -n <namespace>`
+. Verify that your microservice port is named following the requirements requested by Istio: xref:https://istio.io/docs/setup/additional-setup/requirements/[].
+. From the Anypoint Service Mesh installer download location in the Command window, type the following command to bind the adapter with an application:
++
+[source,text,linenums]
+----
+./service-mesh adapter binding create \
+--name=<binding name> \
+--namespace=<namespace name> \
+--adapter=<adapter name> \
+--apiId=<api ID> \
+--serviceName=<service name>
+----
+Note that all objects names, such as adapter name, binding name, and service name, must all be in lower case. An error is thrown if you do not follow Kubernetes naming conventions.
++
+[%header%autowidth.spread]
+|===
+| Parameter Name | Description | Required/Optional | Default Value
+| name | The unique name that represents the binding of the Kubernetes Service with the API, for example, service-binding-anypoint-security”. 
+ | Required | N/A
+| namespace | The namespace in which your service is deployed. Ensure that you use the same namespace as the adapter you provisioned in the previous procedure. | Required | N/A
+| adapter | Specify the name of the adapter that you provisioned in the previous procedure.  | Required | N/A 
+| apiId | Specify the API ID of the API that you want to associate with this adapter. Use the API ID you obtained as part of Review Prerequisites section of this document. | Required | N/A.
+| serviceName | The name of the service that you want to bind to the API, which will be managed by Anypoint Platform. The service name is the identifier of the service that you want to manage with Anypoint Service Mesh. 
+| Required | N/A.
+|===
++
+The binding becomes activate within a few seconds and the API is successfully registered to the proxy.
+. Verify the binding status using the following command:
++
+`./service-mesh binding list`
++
+After you bind your API, the API in API Manager is marked as Active. You are now ready to apply policies to protect your APIs.
++
+image::api-manager-status-for-service-mesh.png[85%, 85%]
+
+
+[[bind_using_CRD]]
+== Binding the Service Using a CRD
+
+To bind the service using a CRD:
+
+. Create your binding definition file and save it with a '.yaml' file extension:
+. If you created your API using Anypoint Platform, use spec.parameters.apiId:
++
+[source,text,linenums]
+----
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceBinding
+metadata:
+  name: <binding name> # example: orders-api-binding
+  namespace: <namespace name> # example: default
+  labels: 
+    service-mesh.mulesoft.com/kind: adapter-binding
```

Comment:
@tbruno this seems wrong, you shouldn't need both api and adapter binding label

Reply:
Yep, its wrong. In this case is adapter binding. Making suggestion to delete the second line.

Reply:
Done! Thanks!